docs/content/latest/manage/backup-restore/point-in-time-restore-ycql.md

```diff
@@ -276,13 +282,154 @@ Relative times can be in any of the following formats (again, note that you can
+
+    ```sql
+    ycqlsh:pitr> select * from employees;
+    ```
+
+    ```output
+     employee_no | name           | department | salary
+    -------------+----------------+------------+--------
+            1223 |   Lucille Ball | Operations |  70000
+            1224 | John Zimmerman |      Sales |  60000
+            1221 |     John Smith |  Marketing |  50000
+            1222 |    Bette Davis |      Sales |  55000
+
+    (4 rows)
+    ```
+
+#### Undo DROP COLUMN
+
+1. Using the same keyspace as the previous scenarios, alter your table by dropping a column.
+
+    ```sql
+    alter table pitr.employees drop salary;
+    select * from pitr.employees;
+    ```
+
+    ```output
+     employee_no | name           | department
+    -------------+----------------+-----------
+            1223 |   Lucille Ball | Operations
+            1224 | John Zimmerman |      Sales
+            1221 |     John Smith |  Marketing
+            1222 |    Bette Davis |      Sales
+
+    (4 rows)
+    ```
+
+1. Now restore back to a time before this table was altered, as in [Restore from a relative time](#restore-from-a-relative-time).
+
+1. Due to a ycqlsh caching issue, to see the effect of this change, you will need to drop out of your current ycqlsh session and log back in.
+
+1. Check to see that the salary column is back.
+
+    ```sql
+    ycqlsh:pitr> select * from employees;
+    ```
+
+    ```output
+     employee_no | name           | department | salary
+    -------------+----------------+------------+--------
+            1223 |   Lucille Ball | Operations |  70000
+            1224 | John Zimmerman |      Sales |  60000
+            1221 |     John Smith |  Marketing |  50000
+            1222 |    Bette Davis |      Sales |  55000
+
+    (4 rows)
+    ```
+
+### Undo index creation
+
+1. Create an index on the table from the previous examples.
+
+    ```sql
+    use pitr;
+    create index t1_index on employees (employee_no);
+    describe index t1_index;
+    ```
+
+    ```output
+    CREATE INDEX t1_index ON pitr.employees (employee_no)
+    WITH transactions = {'enabled': 'true'};
+    ```
+
+1. Now restore back to a time before this index was created.
+
+    **Note**: Due to a ycqlsh caching issue, to see the effect of this change, you will need to drop out of your current ycqlsh session and log back in.
+
+    ```sh
+    ./bin/ycqlsh -e 'use pitr; describe table pitr.t1_index;'
+    ```
+
+    ```output
+    <stdin>:1:Column family u't1_index' not found
+    ```
+
 ## Limitations
 
-This is a BETA feature, and is in active development. Currently, you can **restore data only**. The feature doesn't support metadata; in other words, rolling back past operations such as CREATE, ALTER, TRUNCATE, and DROP TABLE is unsupported.
+This is a BETA feature, and is in active development. Currently, you can recover from the following operations:
+
+* Data changes
+* CREATE INDEX
+* CREATE TABLE
+* ALTER TABLE (including ADD and DROP COLUMN)
+* DROP TABLE
```

Comment:
this we don't have docs for yet, but technically support just landed, so we could try out an example, maybe with both dropping a table and an index